apps/src/templates/teacherNavigation/LessonMaterialsContainer.tsx

```diff
@@ -141,6 +145,8 @@ const LessonMaterialsContainer: React.FC = () => {
         unitNumber={unitNumber}
         lessonNumber={selectedLesson.position}
         resources={selectedLesson.resources.Student}
+        standardsUrl={null}
```

Comment:
I wanted to intentionally pass in `null` here since this is for `studentResources` which shouldn't be shown the standards or vocabulary. 

As you can see, I am kind of waffling between explicitly passing in something as `null` and creating optional props.  It is possible that this is another case where these should be optional and I should just check if they are undefined or not instead.  Interested in your thoughts, Dave.

Reply:
good question! it would be slightly more conventional to make the prop optional. if there's a practical reason that passing null is preferred, I'm interested in hearing more about that (apologies I have not dug into the code review yet)

Reply:
I ended up making this optional.